### ISSUE TITLE
Training page sidebar text does not match content

### ISSUE THREAD

**Issue by maurizi** [azavea/nyc-trees#576]
When all online training is complete, the main content show "Training Complete", but the sidebar indicates there is more training to be done
![almost_there](https://cloud.githubusercontent.com/assets/4432106/6219191/52580fce-b5f4-11e4-993f-cfbe2c19e82f.png)

**Comment by dboyer** [comment]
Let's leave this for now as we're waiting on text from Parks and TreeKIT.

**Comment by jfrankl** [comment]
Will need a developer to write conditional statements to show correct text based on conditions.

**Comment by steventlamb** [comment]
Changing back to design due to #1143

**Comment by steventlamb** [comment]
Assigning to @dboyer because we're just waiting on content.

**Comment by dboyer** [comment]
Is this a duplicate of #443 ?
